package.json

```diff
@@ -32,8 +32,9 @@
         "async": "^2.6.1",
         "base58-native": "^0.1.4",
         "bignum": "^0.13.0",
-        "bitcoinjs-lib-zcash": "git+https://github.com/s-nomp/bitcoinjs-lib.git",
+        "bitgo-utxo-lib": "git+https://github.com/miketout/bitgo-utxo-lib.git",
```

Comment:
@nettts I saw you cloned that into this repo. If we don't plan to use it, I would suggest we delete the fork to minimize confusion. We should also *archive* bitcoinjs-lib.